### PR TITLE
Remove access to Antonio

### PR DESCRIPTION
**Pull request by sauloperez**

### CODE REVIEW

File: inventory/host_vars/local.somconnexio.coop/config.yml [hunk 1]
```diff
@@ -13,9 +13,6 @@ system_administrators:
   - name: enrico
     ssh_key: '{{ inventory_dir }}/../file/keys/enrico.pub'
     state: present
-  - name: antonio
-    ssh_key: '{{ inventory_dir }}/../file/keys/antonio.pub'
-    state: present
```
Comment (enricostano) [inline inventory/host_vars/local.somconnexio.coop/config.yml]: Why this wasn't `absent`? Has the key been removed?

Comment (sauloperez) [reply inventory/host_vars/local.somconnexio.coop/config.yml]: we actually forgot to set it absent but I don't think it matters much in development, does it?

Comment (enricostano) [reply inventory/host_vars/local.somconnexio.coop/config.yml]: OK for development, just worried about the process

Comment (sauloperez) [reply inventory/host_vars/local.somconnexio.coop/config.yml]: thanks for the heads-up yes, we still tend to forget about the state :see_no_evil:

**Pull request merged**